src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/ControllerExtensions.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Localization;
+using OrchardCore.Data;
+using OrchardCore.Environment.Shell;
+using OrchardCore.Tenants.ViewModels;
+
+namespace OrchardCore.Tenants.Controllers
+{
+    internal static class ControllerExtensions
```

Comment:
```suggestion
    public static class ControllerExtensions
```

If you're moving these too extensions, just make them public, it's annoying when you need your own api controller to have to copy all this stuff, just to get validation correct.

Reply:
We could make them `public`, in this case I need to create another PR to make the other extensions public too, such as

https://github.com/OrchardCMS/OrchardCore/blob/acff86890335aaa078ec5afc41a9733d080f8460/src/OrchardCore.Modules/OrchardCore.Users/Controllers/ControllerExtensions.cs#L22

Reply:
Seems there's only one place that I fix it in #11002